armory/scenarios/poisoning_dlbd.py

```diff
@@ -69,7 +70,11 @@ def poison_dataset(self, x, y, return_index=False, fraction=None):
         return poison_x, poison_y
 
 
-class Poison(Scenario):
```

Comment:
Let's keep the name of this `Poison`, so that it's clear that it's the base class that other poisoning scenarios inherit from. You can keep the "dirty label" as part of the docstring, though.